spec/roles_spec.rb

```diff
@@ -9,7 +9,7 @@
     model_instance.stub(:roles=)
   end
 
-  its(:model_instance) { should equal(model_instance) }
+  it { subject.model_instance.should equal(model_instance) }
```

Comment:
This is kinda ugly as the the `it { ... }` form is meant to be used with an implicit `subject` receiver. I would propose to change it into a regular `it "..." do ... end` construct.